WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Pages/PagesCardViewModel.swift

```diff
@@ -232,7 +232,7 @@ private extension PagesCardViewModel {
     func sync() {
         isSyncing = true
         let filter = filter
-        DashboardPostsSyncManager.shared.syncPosts(blog: blog, postType: .page, statuses: filter.statuses.strings)
+        DashboardPostsSyncManager.shared.syncPosts(blog: blog, postType: .page, statuses: filter.statuses)
```

Comment:
Side note: I'd love to centralize the access to this shared resource into an instance property. It would make testing via test doubles possible, as well as cleaning up the code by having a single access. I'll admit that I haven't looked at the code to see whether there are more accesses than this one, but I think it would be a good design practice to get into. If nothing else, it would make for a smaller diff if we'll ever rename the type 😅

Reply:
Do you mean the `DashboardPostsSyncManager.shared` singleton? I agree that we should avoid using singleton. I presume the reason behind using a singleton here is, `PagesCardViewModel` [is expected to be re-created](https://github.com/wordpress-mobile/WordPress-iOS/blob/23.4/WordPress/Classes/ViewRelated/Blog/Blog%20Dashboard/Cards/Pages/DashboardPagesListCardCell.swift#L93) upon cell reusing but syncing (a.k.a API calls) should continue even though the active view model instance is re-created. However, an alternative implementation could be creating a `DashboardPostsSyncManager` instance at view controller level (or anything that holds onto the cells) and pass the instance into cells for instantiating their view models.